www/components/core/Search.tsx

```diff
@@ -93,20 +96,41 @@ export const Search = ({ descriptors }: SearchParams) => {
     }
   }, [searchTerm]);
 
+  function keyPressHandler(event: React.KeyboardEvent, index: number) {
+    if (event.key == "ArrowDown" && index + 1 < (results?.length ?? 0)) {
+      document.getElementById(`SearchResultItem_Link_${index + 1}`)?.focus();
+      event.preventDefault();
+    } else if (event.key == "ArrowUp" && index >= 0) {
+      const target = document.getElementById(
+        index == 0 ? "Search_Input" : `SearchResultItem_Link_${index - 1}`,
+      );
+      target?.focus();
+      event.preventDefault();
+    }
+  }
```

Comment:
What do you think about looping around on the results when one reaches the last element in the dropdown and presses Down?